ChronoStore/CMakeLists.txt

```diff
@@ -1,14 +1,6 @@
-cmake_minimum_required(VERSION 3.17)
-project(chronostore)
-
 find_package(HDF5 REQUIRED)
 find_package(JSON-C REQUIRED)
 
```

Comment:
json-c needs to be added to spack.yaml

Reply:
> json-c needs to be added to spack.yaml

I believe ```JSON-C``` is a dependency of ```Thallium```. so if my assumption that this repo is loaded as a whole stands, that dependency is implied already.